transactor/src/main/scala/io/mediachain/transactor/JsonSerialization.scala

```diff
@@ -0,0 +1,64 @@
+package io.mediachain.transactor
+
+import io.mediachain.transactor.Types._
+import org.json4s._
+
+case class IPLDReference(link: String) extends Reference
```

Comment:
this should have an internal binary representation for storage efficiency reasons